src/main/java/io/numaproj/numaflow/reducestreamer/model/Datum.java

```diff
@@ -0,0 +1,30 @@
+package io.numaproj.numaflow.reducestreamer.model;
```

Comment:
Are these files generated from the same reduce proto?

Reply:
no, I manually created them.

Reply:
We are not generating code from proto files? If that's the case, we should use protoc to generate.

Reply:
Check the init commit https://github.com/numaproj/numaflow-java/commit/67170fcdc26d7f3191552945a701748ee52e07e0.

Reply:
We generate code from proto files. All the POJOs directly communicating through gRPC are auto-generated from proto. We have separate POJOs that users directly work with, those are manually created.